CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- Add missing directory change step in PR instructions
```

Comment:
I think no need to changelog. We can add skip changelog

Reply:
ah, I don't know we can skip changlog. I add this just to pass the check.